sycl/test-e2e/Matrix/Legacy/XMX8/element_wise_all_ops_bf16.cpp

```diff
@@ -7,9 +7,8 @@
 //===----------------------------------------------------------------------===//
 // REQUIRES: matrix-xmx8
 
-// RUN: %clangxx -fsycl %s -o %t.out -DSYCL_EXT_ONEAPI_MATRIX_VERSION=1
-// RUN: %CPU_RUN_PLACEHOLDER %t.out
-// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %{build} -o %t.out -DSYCL_EXT_ONEAPI_MATRIX_VERSION=1
+// RUN: %{run} %t.out
```

Comment:
How would ACC run be excluded?

Reply:
That should be ensured by the programmer who runs `llvm-lit` via `matrix-xmx8` above. We don't auto-detect it, so it would be purely human responsibility.

Reply:
okay